common/Resources/ti.internal/extensions/node/buffer.js

```diff
@@ -1644,7 +1645,6 @@ function setAdjustedIndex(buf, index, value) {
 	if (index >= 0 || index < buf._tiBuffer.length) {
 		buf._tiBuffer[index + buf.byteOffset] = value;
```

Comment:
We should return true in this case to indicate success setting the value, and return false below (outside the if) to indicate failure (out of bounds)